common/app/common/pages/PageHeader.java

```diff
@@ -0,0 +1,31 @@
+package common.pages;
+
+import common.cms.CmsPage;
+
+public class PageHeader {
```

Comment:
It may be more flexible to have just CmsPage and some container with all CMS elements and helpers to extract data by name/code.